frontend/js/main.js

```
@@ -289,8 +286,13 @@ class ApiClient {
   }
 
   static async getTodayOrdersSummary(date) {
-    const dateParam = date ? `?date=${date}` : '';
-    return this.request(`/orders/summary${dateParam}`);
+    const endpoint = date ? `/orders/summary?date=${date}` : '/orders/summary';
+    return await this.request(endpoint);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)